src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/model/sar/SARLicenceData.kt

```diff
@@ -0,0 +1,138 @@
+package uk.gov.justice.digital.hmpps.hmppshdcapi.model.sar
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.Approval
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.BespokeCondition
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.Cas2Referral
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.Curfew
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.Document
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.Eligibility
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.FinalChecks
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.Licence
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.LicenceData
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.LicenceVersion
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.ProposedAddress
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.Reporting
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.Risk
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.Victim
+import uk.gov.justice.digital.hmpps.hmppshdcapi.model.sar.SARConditionFormatter.getConditionText
+import uk.gov.justice.digital.hmpps.hmppshdcapi.model.sar.SARConditionFormatter.getConditionValues
+import java.time.LocalDateTime
```

Comment:
In order to provide a different view of additional conditions, I've created new representations to represent the existing entities .. rather than map every single part of the json structure I've reused DTOs and just overridden the condition part with a new representation. 

This provides a seam to customise the SAR response further if necessary